docs/learn/advanced/security.md

```diff
@@ -0,0 +1,49 @@
+# Security
```

Comment:
This page is missing the vulnerability/concerns that were discussed in https://github.com/ceramicnetwork/js-ceramic/pull/666